lib/src/swipe_detector.dart

```diff
@@ -95,6 +95,7 @@ class SwipeDetector extends StatefulWidget {
     this.onSwipeDown,
     this.onSwipeLeft,
     this.onSwipeRight,
+    this.updatable = false,
```

Comment:
Maybe `liveFeedback`  would be a better name. And please also update the documentation of the constructor.